### ISSUE TITLE
Two finger swipe up moves to the back button.

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/a11y-research#152]
Found during https://github.com/phetsims/QA/issues/471. Confirmed that it occurred in a published sim as well (GFLB).
From slack:
>Kathryn Woessner:office:  3:45 PM
Odd thing I just noticed, think it might have come from a new iPadOS update. Swiping up with two fingers on mobile VO doesn't move you to the top of the scene description, it moves you up to the back arrow and reads through the url, tabs, etc. Is this now expected/something we care about/can fix?
3:45
If we do want to address it, where should the issue go?
Jesse Greenberg  3:46 PM
hmm, most likely something we cannot address, but I would like to know a little more, can you please put an issue in a11y-research so I can look into it?
Kathryn Woessner:office:  3:47 PM
Ok.

I was unable to find a way to go to just the top of the scene description while doing the test, but it may still be possible.

**Comment by jessegreenberg** [comment]
Thanks for pointing out @KatieWoe - I was able to make this happen for a few sims. I also made it happen for a few web pages outside of PhET. This is just the behavior of VoiceOver now and is very much out of our control.

Still helpful to know this is how it works. Closing.

**Comment by zepumph** [comment]
@jessegreenberg do you think this counts as an a11y "quirk" to track?

**Comment by jessegreenberg** [comment]
The Apple description for this in https://support.apple.com/guide/iphone/learn-voiceover-gestures-iph3e2e2281/ios is "Speak the entire screen from the top", so I think this is the intended behavior.
